build/config-builder.js

```diff
@@ -42,6 +42,9 @@ const BASE_CONFIG = {
 
   // Whether or not the build is packaged in an electron distributable
   packaged: false,
+
+  // The Google Analytics tracking identifier to use when instrumenting.
+  google_analytics_UA: 'UA-76122102-1'
```

Comment:
I have no particularly strong opinion.  I see that Atom includes there tracking identifier in source https://github.com/atom/metrics/blob/master/lib/reporter.coffee#L109.  It wouldn't be hard to extract from the code.  This is just a personal test account tracking identifier; let's land it as is, and hide it if necessary when we move to a Mozilla-funded account.